### DISCUSSION TITLE
k8s pod replication

### DISCUSSION THREAD

**Discussion by ghost** [OrchardCMS/OrchardCore#15773] [Q&A] [answered]
When using pod replication in kubernetes, my settings "themes etc" only get applied to the current pod, but not others. Any advice on what I should change?

**Comment by Piedone** [comment, marked as answer]
You'll need to use a common database (i.e. not a local SQLite), shared Media storage (like a shared drive, [Blob Storage](https://docs.orchardcore.net/en/latest/docs/reference/modules/Media.Azure/) or [Amazon S3](https://docs.orchardcore.net/en/latest/docs/reference/modules/Media.AmazonS3/)), and something like [Redis](https://docs.orchardcore.net/en/latest/docs/reference/modules/Redis/) for multi-node hosting.

**Reply by ghost** [reply]
I'm using a sql server db currently and azure blob storage for media.  I'm using the default configuration otherwise.  I'll try adding Redis to see if that resolves the issue

**Reply by ghost** [reply]
You were right.

**Reply by Piedone** [reply]
I'm glad it helped!
